src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -125,7 +132,10 @@ export function useRefetchQuoteCallback() {
           price,
           chainId,
           lastCheck: Date.now(),
-          fee
+          fee,
+          // if we get a fee back but no price, its a fee issue
+          // otherwise it would be a handled api error
+          feeExceedsPrice: Boolean(!price && fee)
```

Comment:
We have the amount, and we have the fee. Shouldn't this be based on the `amout-fee <= 0` instead of assuming that if price is null, is because fees a re bigger than amount?

Reply:
fixed in https://github.com/gnosis/gp-swap-ui/pull/645#discussion_r635168532